<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1200</width>
    <height>812</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Renderer</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QVBoxLayout" name="verticalLayout_graphicsView">
      <item>
       <widget class="QGraphicsView" name="graphicsView_image">
        <property name="styleSheet">
         <string notr="true">background-color: rgb(121, 134, 255);</string>
        </property>
        <property name="sizeAdjustPolicy">
         <enum>QAbstractScrollArea::SizeAdjustPolicy::AdjustToContents</enum>
        </property>
        <property name="renderHints">
         <set>QPainter::RenderHint::Antialiasing|QPainter::RenderHint::LosslessImageRendering|QPainter::RenderHint::TextAntialiasing</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="horizontalLayout_3">
        <item>
         <widget class="QLabel" name="label_vertexCountLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string>Vertices:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_vertexCount">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="Line" name="line">
          <property name="styleSheet">
           <string notr="true">background-color: rgb(85, 0, 255);</string>
          </property>
          <property name="lineWidth">
           <number>1</number>
          </property>
          <property name="orientation">
           <enum>Qt::Orientation::Vertical</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_polygonCountLabel">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string>Polygons:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_polygonCount">
          <property name="styleSheet">
           <string notr="true">color: rgb(56, 66, 255);</string>
          </property>
          <property name="text">
           <string>0</string>
          </property>
         </widget>
        </item>
        <item>
         <spacer name="horizontalSpacer_2">
          <property name="orientation">
           <enum>Qt::Orientation::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QWidget" name="controlpanel" native="true">
      <property name="maximumSize">
       <size>
        <width>350</width>
        <height>16777215</height>
       </size>
      </property>
      <property name="styleSheet">
       <string notr="true"/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QLineEdit" name="textinp_filename">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="accept_filename">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(151, 174, 255);</string>
         </property>
         <property name="text">
          <string>Load file</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_scale">
         <property name="minimumSize">
          <size>
           <width>0</width>
           <height>0</height>
          </size>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>11</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="text">
          <string>Scale:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="textinp_scale">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="maximum">
          <number>999999</number>
         </property>
         <property name="value">
          <number>200</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLabel" name="label_offset">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>11</height>
          </size>
         </property>
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="text">
          <string>Offset:</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QSpinBox" name="textinp_offset">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="maximum">
          <number>99999999</number>
         </property>
         <property name="value">
          <number>270</number>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QCheckBox" name="checkBox_interactiveRender">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="text">
          <string>Interactive</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <property name="topMargin">
          <number>10</number>
         </property>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_width">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>11</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(56, 66, 255);</string>
             </property>
             <property name="text">
              <string>Width:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="textinp_width">
             <property name="styleSheet">
              <string notr="true">color: rgb(56, 66, 255);</string>
             </property>
             <property name="maximum">
              <number>9999999</number>
             </property>
             <property name="stepType">
              <enum>QAbstractSpinBox::StepType::AdaptiveDecimalStepType</enum>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QWidget" name="verticalWidget_times" native="true">
           <property name="maximumSize">
            <size>
             <width>22</width>
             <height>16777215</height>
            </size>
           </property>
           <property name="font">
            <font>
             <pointsize>13</pointsize>
             <bold>true</bold>
            </font>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_6">
            <property name="sizeConstraint">
             <enum>QLayout::SizeConstraint::SetDefaultConstraint</enum>
            </property>
            <property name="leftMargin">
             <number>3</number>
            </property>
            <item>
             <spacer name="horizontalSpacer">
              <property name="orientation">
               <enum>Qt::Orientation::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>40</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
            <item>
             <widget class="QLabel" name="label_times">
              <property name="maximumSize">
               <size>
                <width>13</width>
                <height>16777215</height>
               </size>
              </property>
              <property name="font">
               <font>
                <pointsize>13</pointsize>
               </font>
              </property>
              <property name="styleSheet">
               <string notr="true">color: rgb(56, 66, 255);</string>
              </property>
              <property name="text">
               <string>x</string>
              </property>
              <property name="margin">
               <number>2</number>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_5">
           <property name="spacing">
            <number>6</number>
           </property>
           <item>
            <widget class="QLabel" name="label_height">
             <property name="maximumSize">
              <size>
               <width>16777215</width>
               <height>11</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">color: rgb(56, 66, 255);</string>
             </property>
             <property name="text">
              <string>Height:</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="textinp_height">
             <property name="styleSheet">
              <string notr="true">color: rgb(56, 66, 255);</string>
             </property>
             <property name="maximum">
              <number>9999999</number>
             </property>
             <property name="value">
              <number>500</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <widget class="QComboBox" name="comboBox_renderType">
         <property name="styleSheet">
          <string notr="true">color: rgb(56, 66, 255);</string>
         </property>
         <property name="editable">
          <bool>false</bool>
         </property>
         <property name="currentText">
          <string>polygons</string>
         </property>
         <property name="placeholderText">
          <string/>
         </property>
         <item>
          <property name="text">
           <string>polygons</string>
          </property>
         </item>
         <item>
          <property name="text">
           <string>vertices</string>
          </property>
         </item>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_render">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(151, 174, 255);</string>
         </property>
         <property name="text">
          <string>Render</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QScrollArea" name="scrollArea_log">
         <property name="widgetResizable">
          <bool>true</bool>
         </property>
         <widget class="QWidget" name="scrollAreaWidgetContents_2_log">
          <property name="geometry">
           <rect>
            <x>0</x>
            <y>0</y>
            <width>338</width>
            <height>346</height>
           </rect>
          </property>
          <layout class="QVBoxLayout" name="verticalLayout_2">
           <item>
            <widget class="QLabel" name="label_log">
             <property name="styleSheet">
              <string notr="true">color: rgb(86, 80, 255);
background-color: rgb(155, 255, 158);</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="textFormat">
              <enum>Qt::TextFormat::PlainText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignTop</set>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="button_save">
         <property name="styleSheet">
          <string notr="true">border-color: rgb(0, 0, 0);
background-color: rgb(151, 174, 255);</string>
         </property>
         <property name="text">
          <string>Save to file</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1200</width>
     <height>27</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
